### PR TITLE
engine: docker-compose manifest and global logs respect the same filters

### PR DESCRIPTION
**Pull request by jazzdan**

### CODE REVIEW

**Comment by jazzdan** [comment]
@nicks sorry I should have made more clear: I saw this as a TODO in the code and decided to fix it as I re-familiarize myself with the log code.

File: internal/engine/docker_compose_log_manager.go [hunk 1]
```diff
@@ -148,3 +154,23 @@ func (w DockerComposeLogActionWriter) Write(p []byte) (n int, err error) {
 }
 
 var _ store.Subscriber = &DockerComposeLogManager{}
+
+func shouldFilterDCLog(p []byte) bool {
+	if bytes.Contains(p, []byte("Attaching to")) {
```
[the remark this answers is not in the document]
Comment (jazzdan) [reply internal/engine/docker_compose_log_manager.go]: Yeah should be a "starts with" check?

File: internal/build/test_utils.go [hunk 2]
```diff
@@ -137,7 +137,11 @@ func (f *dockerBuildFixture) startRegistry() {
 		f.t.Fatal(err)
 	}
 
-	err = stdout.WaitUntilContains("listening on", 5*time.Second)
+	if isCircleCI() {
+		err = stderr.WaitUntilContains("listening on", 5*time.Second)
```
Comment (maiamcc) [inline internal/build/test_utils.go]: womp womp. but legit.

I think i'd prefer `combinedOut.WaitUntilContains` and then we wouldn't need this conditional -- any way to do that?

Comment (jazzdan) [reply internal/build/test_utils.go]: Check it out now!

File: internal/build/test_utils.go [hunk 3]
```diff
@@ -126,10 +125,9 @@ func (f *dockerBuildFixture) getNameFromTest() reference.Named {
 
 func (f *dockerBuildFixture) startRegistry() {
 	stdout := bufsync.NewThreadSafeBuffer()
-	stderr := &bytes.Buffer{}
 	cmd := exec.Command("docker", "run", "--name", "tilt-registry", "-p", "5005:5000", "registry:2")
 	cmd.Stdout = stdout
-	cmd.Stderr = stderr
+	cmd.Stderr = stdout
```
Comment (maiamcc) [inline internal/build/test_utils.go]: this should probs have a comment?

**Pull request merged**